### ISSUE TITLE
Loading AWS S3 URL: External package "s3fs.core" not installed

### ISSUE THREAD

**Issue by Charles-Alexandre-Roy** [saulpw/visidata#2508]
**Small description**
I'm try to load s3 buckets with Visidata on MacOS. According to [this link](https://github.com/ajkerrigan/visidata-plugins?tab=readme-ov-file#vds3-open-amazon-s3-paths-and-objects), as of Visidata v2.12dev, that functionality is now part of Visidata and doesn't require a plugin.

I'm not finding much documentation for getting my environment set up so that Visidata can load s3 links, so I've been following the instructions in the plugin. I installed s3fs with `pip install s3fs` and have installed and configured my AWS CLI.

However, whenever I run vd on an s3 URL, e.g. `vd 's3://'` it returns:
```
saul.pw/VisiData v3.0.2
External package "s3fs.core" not installed; run: pip install s3fs
```
If I list the packages installed with pip, I see that `s3fs v2024.3.1` is installed.

Note that if I run `pip3 list`, I only get the following output:
```
Package Version
------- -------
pip     24.2
wheel   0.44.0
```
I don't know if this is an issue, but if I run `which pip`, I see that pip is being run from my Anaconda installation but if I run `which pip3`, it appears that pip3 is being run from my Homebrew installation.

In addition, if I try to install anything with pip3, I get the following message:

```
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try brew install
    xyz, where xyz is the package you are trying to
    install.

    If you wish to install a Python library that isn't in Homebrew,
    use a virtual environment:

    python3 -m venv path/to/venv
    source path/to/venv/bin/activate
    python3 -m pip install xyz

    If you wish to install a Python application that isn't in Homebrew,
    it may be easiest to use 'pipx install xyz', which will manage a
    virtual environment for you. You can install pipx with

    brew install pipx

    You may restore the old behavior of pip by passing
    the '--break-system-packages' flag to pip, or by adding
    'break-system-packages = true' to your pip.conf file. The latter
    will permanently disable this error.

    If you disable this error, we STRONGLY recommend that you additionally
    pass the '--user' flag to pip, or set 'user = true' in your pip.conf
    file. Failure to do this can result in a broken Homebrew installation.

    Read more about this behavior here: <https://peps.python.org/pep-0668/>

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

**Data to reproduce**
N/A

**Steps to reproduce**

Install Visidata with:
`brew install saulpw/vd/visidata`
Install s3fs with:
`pip install s3fs` 
Install and configure the AWS CLI per these instructions:
https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html

**Expected result**

s3 Bucket would load and be navigable within Visidata.

**Actual result with screenshot**

![image](https://github.com/user-attachments/assets/89f3675a-3c9f-4cc3-b923-4bca8d22f3a5)

**Additional context**

- What platform and version are you using (Linux, MacOS, Windows)?
MacOS Sonoma v14.6.1

- Which version of Python?
Python 3.12.2

- Which terminal are you using (for display and input issues)?
iterm2

**Comment by anjakefala** [comment]
Hi @Charles-Alexandre-Roy!

Those instructions make more sense for folks who installed VisiData using pip! SInce you installed VisiData using homebrew, the instructions for installing the dependencies are different.

They are documented here: https://github.com/visidata/homebrew-vd?tab=readme-ov-file#no-module-named-x-errors But perhaps it would also make sense to document them in the install page? 

It's

```
/path/to/homebrew/Cellar/visidata/{ver}/libexec/bin/pip3 install {dependency}
```

So 

```
/path/to/homebrew/Cellar/visidata/{ver}/libexec/bin/pip3 install s3fs
```

And this is because homebrew creates isolated environments for its packages. You need to install dependencies into those environments.

**Comment by Charles-Alexandre-Roy** [comment]
Thanks so much!

I think that adding those instructions (or a link to them) on the install page would be helpful.

Note, I did have the tweak those instructions slightly. pip3 isn't available in `/path/to/homebrew/Cellar/visidata/{ver}/libexec/bin/` so instead I ran the following, which is the correct command to tell pip3 where to install a package:

```
pip3 install --target /path/to/homebrew/Cellar/visidata/{ver}/libexec/bin/ s3fs
```
